shell/platform/windows/window_win32.cc

```diff
@@ -567,9 +574,17 @@ WindowWin32::HandleMessage(UINT const message,
       const bool extended = ((lparam >> 24) & 0x01) == 0x01;
       // If the key is a modifier, get its side.
       keyCode = ResolveKeyCode(keyCode, extended, scancode);
-      const int action = is_keydown_message ? WM_KEYDOWN : WM_KEYUP;
       const bool was_down = lparam & 0x40000000;
+      bool is_syskey = message == WM_SYSKEYDOWN || message == WM_SYSKEYUP;
+      const int action = is_keydown_message
```

Comment:
In this way `action` would be identical to `message`. There is no reason to do this any more.